packages/gatsby-plugin-gatsby-cloud/src/indicator.js

```diff
@@ -0,0 +1,342 @@
+import React from "react"
+import { POLLING_INTERVAL } from "./constants"
+
+function css(strings, ...keys) {
+  const lastIndex = strings.length - 1
+  return (
+    strings.slice(0, lastIndex).reduce((p, s, i) => p + s + keys[i], ``) +
+    strings[lastIndex]
+  )
+}
+
+const Style = () => (
+  <style
+    dangerouslySetInnerHTML={{
+      __html: css`
+        :host {
+          --purple-60: #663399;
+          --gatsby: var(--purple-60);
+          --purple-40: #b17acc;
+          --purple-20: #f1defa;
+          --dimmedWhite: rgba(255, 255, 255, 0.8);
+          --white: #ffffff;
+          --black: #000000;
+          --grey-90: #232129;
+          --radii: 4px;
+          --z-index-normal: 5;
+          --z-index-elevated: 10;
+          --shadow: 0px 2px 4px rgba(46, 41, 51, 0.08),
+            0px 4px 8px rgba(71, 63, 79, 0.16);
+        }
+
+        [data-gatsby-loading-indicator="root"] {
+          font: 14px/1.5 -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+            Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji",
+            "Segoe UI Symbol" !important;
+          background: var(--white);
+          color: var(--grey-90);
+          position: fixed;
+          bottom: 1.5em;
+          left: 1.5em;
+          box-shadow: var(--shadow);
+          border-radius: var(--radii);
+          z-index: var(--z-index-elevated);
```

Comment:
```suggestion
          z-index: var(--z-index-preview-indicator);
```

Something I just fixed in the original loading indicator that should also be applied here